uint8 guidance_law
uint8 waypoint_list_length
float32[] waypoint_list_x
float32[] waypoint_list_y
float32[] waypoint_list_z